src/containers/Login/layouts/authContent.js

```diff
@@ -0,0 +1,31 @@
+import {Image, Text, View} from 'react-native';
+import {Input, PrimaryButton} from '../../../components';
+
+import React from 'react';
+import logo from '../../../../assets/logol.png';
+import styles from '../styles';
+
+const AuthContent = ({islogin, setIsLogin}) => {
+  return (
+    <View style={styles.container}>
+      <View style={styles.logoContainer}>
+        <Image style={styles.image} source={logo} />
+      </View>
+      <View style={styles.inputContainer}>
+        {!islogin && <Input placeholder="Nombre" />}
+        <Input error placeholder="Usuario" />
+        <Input error placeholder="Contraseña" />
+        <PrimaryButton text={islogin ? 'Login' : 'Sign In'} />
+        <Text style={styles.textWhite}>-or-</Text>
+        <PrimaryButton text={islogin ? 'Login' : 'Sign In'} />
+        <Text
+          onPress={() => setIsLogin(!islogin)}
+          style={{...styles.textWhite, ...styles.underlined}}>
+          {islogin ? "Don't you have an account?" : 'Already have an account?'}
+        </Text>
+      </View>
+    </View>
+  );
+};
```

Comment:
I see around 5 places it does check if `isLogin` and not sure if there are some missing checks around this validation, but it looks like trying to re-use the same view for sign-in / sign-up it's getting the view itself a little bit complex. 

I think what could be shared is the layout, but wdyt about having 2 separate views without any of those checks?

Reply:
Making this could allow make changes easier IMO, but not sure what's the preferred practice in React-Native 🤔 

@fernandoamz WDYT?

Reply:
In the next comments, I try to explain the build of this component  

- [ ] [https://github.com/BC-MAR-21-RN/proyecto-denuncia-ciudadana-equipo-01-1/pull/37#discussion_r633846701](url)   
- [ ] [https://github.com/BC-MAR-21-RN/proyecto-denuncia-ciudadana-equipo-01-1/pull/37#discussion_r633855699](url)

Reply:
Now that I corrected that mistake, think that could be in fact less complex as you said, but there is the same comparison I think is not difficult to understand most of them are text that's why I didn't create another component, the creation of an additional component could impact the performance of the app, because actually when I change the screen clicking the button to change login/sign-in just re-render 4 child-components( now that is corrected), but moving to another component will execute a complete render of a new screen and bring the same components that actually are there. I don't know what will be a better practice complex/perform, but if is necessary to change I will.